### PR TITLE
[BM-551] Migrate from HockeyApp to AppCenter

### PR DESCRIPTION
**Pull request by anyashka**
### Ticket

https://netguru.atlassian.net/browse/BM-551
 
### Task Description

 Migrated to App Center.

### CODE REVIEW

File: bitrise.yml [hunk 1]
```diff
@@ -160,4 +170,4 @@ workflows:
                 Workflow|${BITRISE_TRIGGERED_WORKFLOW_ID}
             - buttons: |-
                 View on Bitrise|${BITRISE_BUILD_URL}
-                View on HockeyApp|https://rink.hockeyapp.net/apps/${HOCKEYAPP_APP_ID}
+                View on App Center|https://appcenter.ms/orgs/${APP_CENTER_ORG_NAME_ID}/apps/${APP_CENTER_APP_NAME_ID}
```
Comment (karolpiateknet) [inline bitrise.yml]: You can also update link in ReadMe.
Currently there is a link to HockeyApp - staging

File: bitrise.yml [hunk 2]
```diff
@@ -129,13 +128,24 @@ workflows:
             - scheme: $XCODEBUILD_SCHEME
             - xcodebuild_options: $XCODEBUILD_OPTIONS
 
-  deploy-hockeyapp:
+  install-appcenter:
```
Comment (karolpiateknet) [inline bitrise.yml]: We are missing this step inside https://github.com/netguru/how-we-roll/blob/579c4f9c866f4cf8651983b83ad30d0174240cbd/Mobile/iOS/templates/bitrise.yml#L299
Should it be added there?

File: bitrise.yml [hunk 3]
```diff
@@ -58,7 +58,6 @@ workflows:
       - archive-xcode
     after_run:
       - cache-push
-      - deploy-hockeyapp
```
Comment (karolpiateknet) [inline bitrise.yml]: Why did you resigned from deploying production build?

File: bitrise.yml (hunk 1 above)
Comment (anyashka) [reply bitrise.yml]: Good catch!

File: bitrise.yml (hunk 2 above)
Comment (anyashka) [reply bitrise.yml]: Which exactly step fo you mean?

File: bitrise.yml (hunk 3 above)
Comment (anyashka) [reply bitrise.yml]: Because it was not set up for production previously, so it's unneeded.

File: bitrise.yml (hunk 2 above)
Comment (karolpiateknet) [reply bitrise.yml]: install-appcenter is missing in how-we-roll bitrise.yml template. I'm wondering if we should update this template, if this is necessary step.

Comment (anyashka) [reply bitrise.yml]: Oh, you should use this bitrise.yml for now https://github.com/netguru/how-we-roll/pull/217
as it's not merged yet.

**Pull request merged**